src/main/java/dk/kb/image/TemplateMostUsedColors.java

```diff
@@ -0,0 +1,136 @@
+package dk.kb.image;
+
+import dk.kb.image.model.v1.DominantColorDto;
+
+import java.awt.image.BufferedImage;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Template to extract dominant colors from image.
+ * Implementations need to implement a colorspace to work with.
+ * @param <C> represents the datatype that the specific implementation implements.
```

Comment:
Your param description is true enough, but not very usable. What is the type used for?